cli/src/commands.rs

```diff
@@ -271,13 +271,21 @@ pub(crate) async fn handle_command(
                 })
                 .await?;
 
-            wait_confirmation!(
-                format!(
-                    "Fees: {} sat. Are the fees acceptable? (y/N) ",
-                    prepare_response.fees_sat
-                ),
-                "Payment receive halted"
-            );
+            let fees = prepare_response.fees_sat;
+            let confirmation_msg = match payer_amount_sat {
+                Some(_) => format!("Fees: {fees} sat. Are the fees acceptable? (y/N)"),
+                None => {
+                    let min = prepare_response.min_payer_amount_sat;
+                    let max = prepare_response.max_payer_amount_sat;
+                    let service_feerate = prepare_response.service_feerate;
+                    format!(
```

Comment:
I guess we will need to document the fact that in amountless swap applications need to add the service fee on top of the fees_sat.

Reply:
Done: 542dec789e748cae6da29c4ea516086ef8565785